ToolBox/QuantConnect.ToolBox.csproj

```diff
@@ -242,6 +248,15 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
     <None Include="packages.config" />
+    <Content Include="Visualizer\config.json">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
We typically set this to 'copy if newer' so it doesn't overwrite if manually editted in bin/debug